services/ui-src/src/components/app/AppRoutes.tsx

```diff
@@ -10,6 +10,7 @@ export const AppRoutes = () => {
       <ScrollToTopComponent />
       <Routes>
         <Route path="/" element={<HomePage />} />
+        <Route path="/standard" element={<ReportPageWrapper />} />
```

Comment:
I'm ok with doing this for now for easy debugging/creation of components. Is everyone else ok with it?

Reply:
Yep! I spoke to Britt yesterday about it, we'll just handle that once we have real components out there.